src/ErrorHandler.php

```diff
@@ -434,6 +434,8 @@ private function handleException(\Throwable $exception): void
         try {
             if (null !== $previousExceptionHandler) {
                 $previousExceptionHandler($exception);
+
+                return;
```

Comment:
This is the hearth of the change: if we call down the chain of registered error handler and we don't get any exception thrown, we shouldn't do anything else, so the behavior with and without Sentry should be the same.